hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-common/src/test/java/org/apache/hadoop/mapred/TestJobClientGetJob.java

```diff
@@ -18,15 +18,15 @@
 
 package org.apache.hadoop.mapred;
 
-import static org.junit.Assert.assertNotNull;
-
 import java.io.IOException;
 
 import org.apache.hadoop.conf.Configuration;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
```

Comment:
import order is wrong

Reply:
Hi @ayushtkn . I used Intellij to optimize imports. Can you share what should be ideal order here ? Thanks